packages/tc-web/src/components/tc-img-quote-card/tc-img-quote-card.tsx

```diff
@@ -0,0 +1,91 @@
+import { Component, Prop, h } from "@stencil/core";
+
+@Component({
+  tag: "tc-img-quote-card",
+  styleUrl: "tc-img-quote-card.css",
+  shadow: true,
+})
+export class ImgQuoteCard {
+  @Prop() quote: string;
+  @Prop() quotee: string;
```

Comment:
some of the prop names are really hard to identify what it is used for. could you improve naming (does not matter if the prop names are a bit longer, quoteLabel or so?)